src/java.net.http/share/classes/jdk/internal/net/http/ResponseSubscribers.java

```diff
@@ -281,8 +281,13 @@ public void onSubscribe(Flow.Subscription subscription) {
 
         @Override
         public void onNext(List<ByteBuffer> items) {
+            int size = Utils.remaining(items, Integer.MAX_VALUE);
```

Comment:
Hello Chris, are we intentionally limiting the `max` to `Integer.MAX_VALUE` here? This specific `Utils.remaining` will throw an `IllegalArgumentException` if the total `remaining` content across the `items` exceed that max value. If it is intentional to use this method, should we explicitly handle the `IAE` and rethrow it as `IOException`?

Reply:
There's an overloaded `Utils.remaining` which doesn't constrain the maximum size. Perhaps we could use that?

Reply:
👍 Thanks @jaikiran. Agree. Done.